### DISCUSSION TITLE
part-ii/2019-4/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#55] [Question comments]
Part II 2019 Algebraic Topology: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2019-4/

**Comment by artie2000** [comment]
$H_n(K_n,L_{n-1})=\mathbb{Z}$, and all other groups vanish.

**Reply by Kiolt** [reply]
I think that since $C_0(K_n)=C_0(L_{n-1})$ we get $C_0(K_n,L_{n-1})=0$ and $H_0(K_n,L_{n-1})=0$.

**Reply by artie2000** [reply]
Corrected, thanks!

**Reply by dropdbassett** [reply]
Another way to see this is that (by path connectedness) $H_0(K_n)$ and $H_0(L_{n-1})$ are both generated by the class of any point, and hence the map $\iota_* : H_0(L) \to H_0(K)$ in the Mayer-Vietoris sequence is an isomorphism.

**Comment by artie2000** [comment]
A nice way to think of relative homology is to collapse the space being quotiented by to a point -- here $D^n$ becomes $S^n$.
